subscription/src/test/java/org/killbill/billing/subscription/api/user/TestSubscriptionHelper.java

```diff
@@ -99,6 +99,39 @@ public TestSubscriptionHelper(final SubscriptionBaseInternalApi subscriptionApi,
         this.internalCallContextFactory = internalCallContextFactory;
     }
 
+    public static DateTime addOrRemoveDuration(final DateTime input, final List<Duration> durations, final boolean add) {
```

Comment:
Just to make sure I understand, this code just got moved from bottom of the file to the top, but there was no change, right?

Reply:
Yes, after running the code formatter in Intellij, this method was moved from the bottom of the file to the top. There was no other change.